babel.cfg

```diff
@@ -0,0 +1,13 @@
+# Extract from Python source files and Jinja templates.
```

Comment:
Why do we need both this and `securedrop/babel.cfg`?

Reply:
Cognitive copypasta from `securedrop-client`, where this configuration lives in the root of the repository.  :-)  Deduplicated in e9d1d03e47dbf4ff797ad7faad44793dd73c0bee.